src/v2/perf-utils.js

```diff
@@ -627,9 +642,13 @@ class PUClass {
       'loadEventStart': timing.loadEventStart - timing.navigationStart,
       'loadEventDuration': timing.loadEventEnd - timing.loadEventStart
     };
-    const type = (parseInt(this.env.ES_MAJOR, 10) > 5) ? 'doc' : 'resource';
-    let navEntry = `{ "index" : { "_index" : "${this.env.INDEX_RES}-${indexDay}", "_type" : "${type}" } }\n`;
-    navEntry += JSON.stringify(nt) + '\n';
```

Comment:
navEntry was a string w/ 2 JSON objects, with this change it is an Object?

Reply:
Yep - that is correct. It's all due to the change in the elasticsearch client (to support elastic 7.x) and in this case, the navEntry object is returned to the `getResourceBody` function which in turn sends it to the bulk function in `es-utils.js`. This PR shows the change in that bulk function (here: https://github.com/godaddy/timings/pull/73/files/9de01f49346faea0904496272b117b1a2064ca43#diff-848f96ac5f09dd2ddcfe81440ba58dc12eb54e2ad9c21665e12615d2e1a9bbbeL227) which now takes an array rather than a stringified JSON...